po/tr.po

```diff
@@ -0,0 +1,159 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+msgid ""
+msgstr ""
+"Project-Id-Version: spot\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-06-14 00:00+0200\n"
+"PO-Revision-Date: 2021-06-14 00:00+0200\n"
+"Last-Translator: Automatically generated\n"
+"Language-Team: Yusuf Çınar Özmen <marulumbenaq@gmail.com>\n"
+"Language: Turkish\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=CHARSET\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=INTEGER; plural=EXPRESSION;\n"
```

Comment:
You need to fill in an expression here. `"Plural-Forms: nplurals=2; plural=(n != 1);\n"` should be correct for Turkish.
See https://www.gnu.org/software/gettext/manual/html_node/Plural-forms.html
Tools like [Poedit](https://poedit.net/) should do this for you automatically.